lib/council-server/src/server.rs

```diff
@@ -105,71 +161,100 @@ impl Server {
-                        &mut complete_graph,
+                        complete_graph,
                         reply_channel,
                         change_set_id,
                         dependency_graph,
                     )
-                    .await
-                    .unwrap();
-                }
+                    .await,
+                    RequestDiscriminants::ValueDependencyGraph,
+                ),
                 Request::ProcessedValue {
                     change_set_id,
                     node_id,
-                } => {
+                } => (
                     job_processed_a_value(
                         &self.nats,
-                        &mut complete_graph,
+                        complete_graph,
                         reply_channel,
                         change_set_id,
                         node_id,
                     )
-                    .await
-                    .unwrap();
-                }
-                Request::Bye { change_set_id } => {
-                    job_is_going_away(&mut complete_graph, reply_channel, change_set_id)
-                        .await
-                        .unwrap();
-                }
+                    .await,
+                    RequestDiscriminants::ProcessedValue,
+                ),
+                Request::Bye { change_set_id } => (
+                    job_is_going_away(complete_graph, reply_channel, change_set_id).await,
+                    RequestDiscriminants::Bye,
+                ),
                 Request::ValueProcessingFailed {
                     change_set_id,
                     node_id,
-                } => {
+                } => (
                     job_failed_processing_a_value(
                         &self.nats,
-                        &mut complete_graph,
+                        complete_graph,
                         reply_channel,
                         change_set_id,
                         node_id,
                     )
-                    .await
-                    .unwrap();
+                    .await,
+                    RequestDiscriminants::ValueProcessingFailed,
+                ),
+                Request::Restart => {
+                    debug!(
+                        %management_channel, %management_reply_channel, "found restart request sent to everyone subscribing to management channel: no-op"
+                    );
+                    (Ok(()), RequestDiscriminants::Restart)
                 }
             };
-        }
 
-        Ok(())
+            match result {
+                Ok(()) => match discrim {
+                    RequestDiscriminants::Restart => {
+                        debug!("no-op successful for restart request")
+                    }
+                    discrim => debug!(?discrim, "processing request successful"),
+                },
+                Err(err) => match err {
+                    Error::DependencyDataMissing => {
```

Comment:
In the event that the dependency data is missing (recent panics seen by @sprutton1 @johnrwatson @stack72), we need to tell all pinga instances to restart actively running jobs.